pages/implementations/mapping.md

```diff
@@ -4,14 +4,14 @@ title: Mapping To Rule
 
 ## Implementation Scoping
 
-An implementation does not need to be a one-to-one mapping to ACT-R rules. A single implementation can test multiple ACT-R. In manual test procedures, it is common to test multiple things at once. For instance, a test procedure looking at the language of a page will simultaneously test whether or not the `<html>` element has a `lang` attribute, if the value of the `lang` attribute is valid, and if the value matches the language of the page. These are 3 separate rules, covered in a single test procedure of an implementation.
+An implementation procedure does not need to be a one-to-one mapping to an ACT rule. A single implementation procedure can test multiple ACT rules. In manual test procedures, it is common to test multiple things at once. For instance, a procedure looking at the language of a page will simultaneously test whether or not the `<html>` element has a `lang` attribute, if the value of the `lang` attribute is valid, and if the value matches the language of the page. These are 3 separate rules, covered in a single test procedure of an implementation.
 
 <figure role="figure" aria-label="Single implementation, mapping to two ACT-R Rules">
   <figcaption>Single implementation, mapping to two ACT-R Rules</figcaption>
   <img src="./images/impl-large-scope.svg" alt="">
 </figure>
 
-An tool or methodology can also have multiple implementations that when combined, map to a single ACT-R rule. It is common in automated test tools to split up certain rules, to build up more appropriate remediation advise. While ACT-R may have a single rule to test that all elements with the `img` role have an accessible name, an automated tool may implement testing `img` elements separate from `role="img"`, so that it gives different remediation advise for elements that can have an `alt` attribute, from elements that can't.
+An accessibility tool or methodology can also have multiple implementation procedures that when combined, map to a single ACT rule. It is common in automated test tools to split up certain rules, to build up more appropriate remediation advise. For example there can be a single ACT rule to test that all elements with the `img` role have an accessible name, an automated tool may implement testing `img` elements separate from `role="img"`, so that it gives different remediation advise for elements that can have an `alt` attribute, from elements that can't.
```

Comment:
```suggestion
An accessibility tool or methodology can also have multiple implementation procedures that when combined, map to a single ACT rule. It is common in automated test tools to split up certain rules to build up more appropriate remediation advice. For example there can be a single ACT rule to test that all elements with the `img` role have an accessible name. An automated tool may implement testing `img` elements separate from `role="img"`, so that remediation advice for elements that can have an `alt` attribute will differ from elements that can't.
```